react/javascript/Makefile

```diff
@@ -7,7 +7,7 @@ TS_MESSAGE_FILES = $(patsubst ../../compatibility-kit/javascript/features/%.ndjs
 
 dist/src/styles/cucumber-react.css: src/styles/styles.scss src/styles/react-accessible-accordion.css
 	mkdir -p $(@D)
-	./node_modules/.bin/sass $< > $@
+	../../node_modules/.bin/sass $< > $@
```

Comment:
What about using `npx sass` instead of a relative path to the binary?

Reply:
and/or could we move to the `package.json`, maybe another candidate for using `prepare`?